docs/src/main/tut/testing.md

```diff
@@ -0,0 +1,139 @@
+---
+menu: main
+weight: 350
+title: Testing
+---
+
+## Introduction
+
+Testing http4s `org.http4s.HttpService`'s is straightforward given that it's:
+
+```scala
+    type HttpService[F[_]] = Kleisli[OptionT[F, ?], Request[F], Response[F]]
+```
+
+The documentation explains:
+
+```
+  /**
+    * A [[Kleisli]] that produces an effect to compute an [[OptionT[F]]]] from a
```

Comment:
I'm slightly worried about quoting the scaladoc like that because this will get out-dated quite easily .

Reply:
Heh you're not wrong. #1693 is already changing this.